site/src/component/SideInfo/SideInfo.scss

```diff
@@ -30,6 +30,10 @@
   > *:not(:last-child) {
     margin-bottom: 20px;
   }
+
+  .MuiInputBase-root .MuiInputBase-input {
+    background-color: var(--overlay2);
+  }
```

Comment:
maybe not for this PR since we are in a bit of a crunch to get it merged, but in the future we should define additional variants in the theme that will apply the background color. I've done this for #826 but we can have this as a @todo and/or a new issue